### ISSUE TITLE
When I encountered a problem running 'evaluatelatent_embedder. py'

### ISSUE THREAD

**Issue by Shame-fight** [mueller-franzes/medfusion#21]
When I run evaluate_ Latent_ When embedding. py, the code will automatically download the model, but I have specified a trained VAE model in the code, and I am not sure if this is normal. I am concerned that using the current code to evaluate the pre trained model instead of the model I just trained, may I ask how to resolve this.
`(medfusion_jgx) [root@node3 medfusion]# python scripts/evaluate_latent_embedder.py 
INFO:root:Samples Real: 637
Setting up [baseline] perceptual loss: trunk [vgg], v[0.1], spatial [off]
/root/anaconda3/envs/medfusion_jgx/lib/python3.8/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/root/anaconda3/envs/medfusion_jgx/lib/python3.8/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG16_Weights.IMAGENET1K_V1`. You can also use `weights=VGG16_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Downloading: "https://download.pytorch.org/models/alexnet-owt-7be5be79.pth" to /root/.cache/torch/hub/checkpoints/alexnet-owt-7be5be79.pth
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 233M/233M [00:59<00:00, 4.09MB/s]`
